nautilus_trader/data/aggregation.pyx

```diff
@@ -857,6 +900,7 @@ cdef class TimeBarAggregator(BarAggregator):
 
         return start_time
 
+
```

Comment:
Methods should be separated by a single blank, functions are separated by two blanks.

This would be autoformatted if it was a `.py`, except our Python formatter can't parse this Cython.